kvmagent/kvmagent/plugins/imagestore.py

```
@@ -136,6 +136,55 @@ def _get_progress(synced):
+            node_name = self.get_disk_device_name(target_disk)
+
+            target_install_path = volume_info.target
+            volumes += ",".join([node_name, target_install_path]) + ";"
+
+        PFILE = linux.create_temp_file()
+        with linux.ShowLibvirtErrorOnException(vm):
+            cmdstr = '%s cbtbak -domain %s -volumes "%s" -bitmap "%s" -portrange "%s" > %s' % \
+                     (self.ZSTORE_CLI_PATH, vm.uuid, volumes, bitmapTimestamp, portRange, PFILE)
+            shell.call(cmdstr)
+            with open(PFILE) as fd:
+                linux.rm_file_force(PFILE)
```

Ruff (0.14.7)

`linux` may be undefined, or defined from star imports

(F405)